File: Factory/Java/LogisticsNoFactory/src/LogisticsNoFactory/Coordinate.java
package LogisticsNoFactory;

public class Coordinate {
	private String _name;
	private double _x;
	private double _y;
	
	public Coordinate(String name, double x, double y)
	{
		_name = name;
		_x = x;
		_y = y;
	}
	
	@Override
	public String toString()
	{
		return String.format("%s %.8f %.8f", _name, _x, _y);
	}
	
}

File: Factory/C#/Logistics/NoFactory/Coordinate.cs
namespace NoFactory
{
    public class Coordinate
    {
        private readonly string _name;
        private readonly double _x;
        private readonly double _y;

        public Coordinate(string name, double x, double y)
        {
            _name = name;
            _x = x;
            _y = y;
        }

        public override string ToString()
        {
            return $"{_name} {_x} {_y}";
        }

    }
}